projects/packages/videopress/src/client/block-editor/blocks/video/types.ts

```diff
@@ -1,14 +1,27 @@
-type Track = {
-	items: object;
+import { trackKindOptionProps } from '../../plugins/video-chapters/utils/tracks-editor/types';
+
+export type videoId = number;
+
+type track = {
+	label: string;
+	srcLang: string;
+	kind: trackKindOptionProps;
+	src: string;
+};
+
+export type videoBlockColorAttributesProps = {
+	seekbarPlayedColor?: string;
+	seekbarLoadingColor?: string;
+	seekbarColor?: string;
 };
 
-export type VideoBlockAttributes = {
-	id?: number;
+export type videoBlockAttributes = videoBlockColorAttributesProps & {
```

Comment:
I think the convention we have been using is to use `PascalCase` for types and interfaces and `camelCase` for their properties.

Reply:
ohhh :'(

Reply:
https://github.com/Automattic/jetpack/pull/27419/commits/e8d2275c5bf857b1957147980a5862753aef4876